src/mixins/raster-layer-line-mixin.js

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable complexity */
```

Comment:
we voted this rule out of Immerse long ago, in case the current generation is interested in doing the same for charting...